components/ws-daemon/pkg/content/initializer.go

```diff
@@ -196,11 +200,11 @@ func RunInitializer(ctx context.Context, destination string, initializer *csapi.
 	}
 
 	// TODO(cw): make the initializer work without chown
-	spec.Process.Capabilities.Ambient = append(spec.Process.Capabilities.Ambient, "CAP_CHOWN")
-	spec.Process.Capabilities.Bounding = append(spec.Process.Capabilities.Bounding, "CAP_CHOWN")
-	spec.Process.Capabilities.Effective = append(spec.Process.Capabilities.Effective, "CAP_CHOWN")
-	spec.Process.Capabilities.Inheritable = append(spec.Process.Capabilities.Inheritable, "CAP_CHOWN")
-	spec.Process.Capabilities.Permitted = append(spec.Process.Capabilities.Permitted, "CAP_CHOWN")
+	spec.Process.Capabilities.Ambient = append(spec.Process.Capabilities.Ambient, "CAP_CHOWN", "CAP_FOWNER")
```

Comment:
it is because the initializer runs in own container under gitpod user, correct? Just checking my understanding.

Reply:
It allows to do chmod within the init container. It might no longer be necessary.

Reply:
Yes. We run the content init in its own container (via runc) as non-privileged user. That content init process will need to restore files with original permissions/ownership though, hence these capabilities.